lib/Drupal/drupal_helpers/Field.php

```diff
@@ -19,6 +19,8 @@ class Field {
    *
    * @param string $field_name
```

Comment:
please revert changes to this file as this PR is only about Rules and not Fields.